test/elixir/lib/couch.ex

```diff
@@ -72,4 +131,35 @@ defmodule Couch do
         %HTTPotion.ErrorResponse{ message: error_to_string(reason)}
     end
   end
+
+  # Anther HACK: Until we can get process_request_headers/2 merged
+  # upstream.
+  @spec process_arguments(atom, String.t, [{atom(), any()}]) :: %{}
+  defp process_arguments(method, url, options) do
+    options    = process_options(options)
+
+    body       = Keyword.get(options, :body, "")
+    headers    = Keyword.merge Application.get_env(:httpotion, :default_headers, []), Keyword.get(options, :headers, [])
```

Comment:
Maybe use `(` for consistency? Or does Elixir style encourages not using parentheses in this particular case.

Also maybe shorten to < 80 columns?

Reply:
For the parens vs. no parens, I have yet to distinguish or come up with a general rule of thumb for when to use them. However the number of ways things parse weirdly I'm inclined to say almost never leave them out.

That said, this particular functions i copy pasted from the HTTPotion library and only exists as a copy here so that we can use the process_request_headers/2 callback on line 159. Assuming we can get upstream to take that minor patch we'll be able to get rid of this function copy.

Reply:
Which is to say I'm not inclined to reformat this bit of code so that if we do have to do a diff for any reason in the future that'd cause us issues.

Reply:
Ah alright. Makes sense. Let's keep it.